src/scripts.js

```diff
@@ -110,30 +110,29 @@ function displayHydrationInfo(userInfo, id, hydrationInfo) {
 
 function displaySleepInfo(userInfo, id, sleepInfo) {
   let sleepRepo = new Sleep(sleepInfo);
-  let today = makeToday(userInfo, id, sleepRepo.sleepData);
+  let today = userInfo.getToday(id, sleepRepo.sleepData);
```

Comment:
I wonder if we should create this variable earlier and pass it in where needed to avoid calling it multiple times